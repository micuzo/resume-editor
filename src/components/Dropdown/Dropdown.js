import React from 'react';
import prettyPrint from '../../Js/PrettyPrint';

import classes from './Dropdown.module.css';

const Dropdowm = (props) => {

    let value;
    if(props.value === 'n/a') value = 'N/A';
    else value = prettyPrint(props.value);

    return(
        <select value={value} onChange={(event) => props.update(event)} className={classes.dropdown}>
                <option value='Strong'>Strong</option>
                <option value='Medium'>Medium</option>
                <option value='Familiar'>Familiar</option>
                <option value='N/A'>N/A</option>
        </select>
    );
}

export default Dropdowm;