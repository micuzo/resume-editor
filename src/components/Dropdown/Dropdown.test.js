import React from 'react';
import { mount, render, shallow } from 'enzyme';
import Dropdown from './Dropdown';

describe('Dropdown', () => {
    it('Should render correctly', () => {
        const component = shallow(<Dropdown value='Strong'/>);

        expect(component).toMatchSnapshot();
    });
})