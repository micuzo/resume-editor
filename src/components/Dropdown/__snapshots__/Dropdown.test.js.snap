// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dropdown Should render correctly 1`] = `ShallowWrapper {}`;
